src/compiler/test-file/formats/typescript/compiler.js

```diff
@@ -17,14 +17,14 @@ export default class TypeScriptTestFileCompiler extends APIBasedTestFileCompiler
             allowJs:                 true,
             pretty:                  true,
             inlineSourceMap:         true,
-            noImplicitAny:           false,
+            noImplicitAny:           true,
```

Comment:
We shouldn't enable strict type checking options until we provide to a user a way to disable them.